2020-02-Spring/1_strings/README.md

```diff
@@ -97,21 +117,70 @@ Go to [Solution](#s3)   [Top](#top)
 <!-- Don't remove -->
 <a name="s1"/>
 
-### 1. SOLUTION 1 TODO :bug:
+### 1. Reverse String
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/reverse-string/solution/)
+ 
+#### Optimal Solution
 
-#### Naive/Simple Solution
+<details>
+<summary>Click to see solution</summary>
```

Comment:
The summaries are good. IMO, only the code should be hidden with the dropdown; I personally would leave the algorithm and written explanation out of the dropdown. (This is just a personal opinion -- talk with the other collaborators to see what they think.)